pyproject.toml

```diff
@@ -83,10 +83,7 @@ known-local-folder = ["src"]
 'content-type' = "text/markdown"
 
 [project.scripts]
-mochada_kit = "mochada_kit.__init__:main"
-
-[tool.setuptools.dynamic]
-version = { attr = "mochada_kit.__version__" }
+mochadakit = "mochada_kit.cli:cli"
```

Comment:
I'd like to have the name of the project script as "mochada_kit" with the hyphen, just to keep it the same as the package name.